### ISSUE TITLE
Portal: Download the manifest file from the portal

### ISSUE THREAD

**Issue by gokul472** [overture-stack/roadmap#81]

**Comment by gokul472** [comment]
This is nice to have ticket.

**Comment by b-f-chan** [comment]
Work on the download button options (both TSV, manifest) need to be unblocked by the Arranger dependency issue first (arranger#644)
